worker/create_batches_from_filters_worker.go

```diff
@@ -53,6 +54,20 @@ type CreateBatchesFromFiltersWorker struct {
 	RedisClient               *redis.Client
 }
 
+// PrintMemUsage ...
+func PrintMemUsage() {
+	var m runtime.MemStats
+	runtime.ReadMemStats(&m)
+	// For info on each, see: https://golang.org/pkg/runtime/#MemStats
+	fmt.Printf("Alloc = %v MiB", bToMb(m.Alloc))
```

Comment:
Even if this is for debug issues I'd rather use the logger instead of `fmt.Printf`.